grails-app/views/project/_details.gsp

```diff
@@ -1,434 +1,506 @@
+                                                <li class="nav-header" style="display:none;" data-bind="visible:userOrganisationResults().length && otherResults().length">Other organisations</li>
+                                                <!-- ko foreach:otherResults -->
+                                                <li data-bind="css:{active:$parent.isSelected($data)}"><a data-bind="click:$parent.select, text:name"></a></li>
+                                                <!-- /ko -->
+                                            </ul>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="row-fluid" data-bind="visible: $parent.transients.associatedOrgNotInList()">
+                                <div class="span3"></div>
+
+                                <div class="span9">
+                                    <div class="clearfix control-group">
+                                        <label class="control-label left-aligned-label span3" for="associatedOrgUrl"><g:message code="project.details.associatedOrgs.url"/></label>
+
+                                        <div class="controls span12 margin-left-0">
+                                            <input id="associatedOrgUrl" class="input-xxlarge" type="text" data-bind="value: $parent.transients.associatedOrgUrl">
+                                        </div>
+                                    </div>
+                                    <div class="clearfix control-group">
+                                        <label class="control-label left-aligned-label span3" for="associatedOrgLogo"><g:message code="project.details.associatedOrgs.logo"/></label>
+
+                                        <div class="controls span12 margin-left-0">
+                                            <input id="associatedOrgLogo" class="input-xxlarge" type="text" data-bind="value: $parent.transients.associatedOrgLogoUrl">
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
 
-            <div class="controls span9">
-                <fc:datePicker class="input-small" targetField="plannedEndDate.date" name="plannedEndDate"
-                               id="plannedEndDate" data-validation-engine="validate[future[plannedStartDate]]"/>
-                <g:message code="project.details.plannedEndDate.extra"/>
+                    <div class="row-fluid">
+                        <div class="span3"></div>
+                        <div class="span9">
+                            <a href="#" data-bind="click: addSelectedOrganisation, visible: term" class="margin-top-2"><i class="fa fa-check">&nbsp;</i><g:message code="project.details.associatedOrgs.add"/></a>
```

Comment:
for consistency with rest of the page, it might be better to have this as a button.